wordle-web/components/available-wordlist.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-card class="my-5 pa-5">
+    <v-list style="max-height:200px" class="overflow-y-auto overflow-x-hidden">
+    <v-row>
+      <v-container class="text-left" style="position:absolute">
+        Possible: {{wordleGame.viewHints.length === undefined? 0 : wordleGame.viewHints.length}}
+      </v-container>
+    </v-row>
+    <v-row v-for="(word, i) in wordleGame.viewHints" :key="i" no-gutters justify="center">
+      <v-container class="text-center">
+        <v-btn :disabled="wordleGame.gameOver" @click="setWord(word)"> {{ word }}</v-btn>
+      </v-container>
```

Comment:
The long list of available words at the bottom of the screen requires a lot of scrolling on the page, try wrapping the <v-btn> that has {{ word }} with some <v-col> tags, should give you more button words on a line.